package.json

```diff
@@ -8,9 +8,9 @@
         "build:watch": "tsc --watch",
         "test": "jest --config=jest.config.js --detectOpenHandles --silent",
         "test:watch": "jest --watch --notify --config=jest.config.js --detectOpenHandles --silent",
-        "prettier:check": "prettier --check './**/*.{js,ts,json,md}'",
-        "prettier:write": "prettier --write './**/*.{js,ts,json,md}'",
-        "prepare": "npm run build"
```

Comment:
Is het runnen van de build nog een belangrijk onderdeel in het proces, husky gooit dit nu weg. Maar moet wellicht onderdeel zijn van een hook.

Reply:
ok heb hem nu ook aan prepublish hook toegevoegd.